src/pages/resource/description/Project.tsx

```diff
@@ -0,0 +1,147 @@
+import { Button, ListItem, List, TextField as MuiTextField } from '@material-ui/core';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import CristinProjectType from '../../../types/cristin_project.types';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { searchCristinProjects } from '../../../api/cristinProject';
+
+const StyledCristinProject = styled.div`
+  margin: 0.5rem;
+  padding-bottom: 1rem;
+`;
+
+const StyledInputBox = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: 0.3rem;
+`;
+
+const StyledFieldHeader = styled.header`
+  font-size: 1.5rem;
+`;
+
+const StyledMuiTextField = styled(MuiTextField)`
+  margin-right: 1rem;
+`;
+
+const StyledResultBox = styled.div`
+  background-color: ${({ theme }) => theme.palette.background.default};
+  border: 1px solid grey;
+  padding: 1rem;
+`;
+
+const StyledSelectedProjectList = styled.div`
+  padding: 1rem;
+`;
+
+const StyledResultListBox = styled.div`
+  padding: 0.4rem;
+`;
+
+const StyledDeleteIcon = styled(DeleteIcon)`
+  color: red;
+`;
+
+const Project: React.FC = () => {
+  const { t } = useTranslation();
+  const [titleValue, setTitleValue] = useState('');
+  const [idValue, setIdValue] = useState('');
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const [projectList, setProjectList] = useState<CristinProjectType[]>();
+  const [selectedProjectList, setSelectedProjectList] = useState<CristinProjectType[]>([]);
+
+  const handleProjectSearch = async () => {
+    let projects = [];
+    if (idValue) {
+      projects = await searchCristinProjects(`id=${idValue}`);
+      setSearchTerm(idValue);
+    } else if (titleValue) {
+      projects = await searchCristinProjects(`title=${titleValue}`);
+      setSearchTerm(titleValue);
+    }
+    projects ? setProjectList(projects) : setProjectList([]);
+    setTitleValue('');
+    setIdValue('');
+  };
+
+  const handleTitleValueChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitleValue(event.target.value);
+  };
+
+  const handleIdValueChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIdValue(event.target.value);
+  };
+
+  const handleListItemClick = (event: any, project: CristinProjectType) => {
+    setSelectedProjectList([...selectedProjectList, project]);
+    setProjectList(undefined);
+  };
+
+  const removeProject = (projectToDelete: CristinProjectType) => {
+    setSelectedProjectList(selectedProjectList.filter(project => projectToDelete !== project));
+  };
+
+  return (
+    <StyledCristinProject>
+      <StyledFieldHeader>{t('resource_form.project_assosiation')}</StyledFieldHeader>
+      {selectedProjectList.length > 0 ? (
+        <StyledSelectedProjectList>
+          <List>
+            {selectedProjectList.map((project: CristinProjectType, index: number) => (
+              <ListItem key={index}>
```

Comment:
bruke `cristin_project_id` som key? https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c